### ISSUE TITLE
Some questions about reproduce model.

### ISSUE THREAD

**Issue by chen0z** [xuezhang335/DeepTAP#1]
Hello, I'm a graduate student currently delving into this paper.

To better understand your work and replicate your results, I have a few questions regarding the training details of the model:

1. Which optimizer did you use (e.g., Adam, SGD...) and what were the related parameters (e.g., learning rate...)?
2. Did you employ a scheduler to adjust the learning rate, and what were the relevant parameters (e.g., step...)?
3. Did you implement early stopping, and what were the related parameters (e.g., patience=10)?

If you could provide the training code, that would be even better.
